build.sbt

```diff
@@ -177,5 +194,6 @@ lazy val root = (project in file("."))
   .aggregate(wdl)
   .aggregate(wom)
   .aggregate(womtool)
+  .aggregate(languageFactoryCore)
```

Comment:
Add all the new sub-projects to aggregate. Otherwise they will not be part of the root build.

For example right now they will NOT be tested during `sbt test`.

Reply:
Done